infra/skypilot/README.md

```diff
@@ -0,0 +1,33 @@
+Loosely based on [these docs](https://skypilot.readthedocs.io/en/latest/examples/gpu-jupyter.html#:~:text=SkyPilot%20makes%20the%20process%20of,managing%20provisioning%20and%20port%20forwarding.&text=View%20the%20supported%20GPUs%20with%20the%20sky%20show%2Dgpus%20command.&text=Enter%20the%20password%20or%20token,can%20create%20a%20new%20notebook.).
```

Comment:
In the README, using `exit` didn't seem to do anything. Any chance you could write separate instructions for each of the terminals? (i.e the one you type sky launch and the one you SSH into). 

Q: What is the different between sky down and sky stop? Options showed up when I pressed control+c to get out of the instance after `exit` didn't work.

Reply:
Oh interesting ok 🤔

Hm I think it's like the difference between putting your computer to sleep, and shutting your computer down. From [these docs](https://skypilot.readthedocs.io/en/latest/reference/cli.html?highlight=sky%20down#sky-stop) it seems like you can't use `sky stop` if you have a spot instance (which is what we're using because it's cheaper). `stop` preserves the data that was on the cluster, while `down` completely gets rid of the cluster and any data you had on it.

Reply:
And yes I'll write the instructions for the separate terminals 👍🏻